src/SmartComponents/BaselinesTable/BaselinesTable.js

```diff
@@ -312,36 +234,31 @@ BaselinesTable.propTypes = {
     baselineTableData: PropTypes.array,
     createBaselinesTable: PropTypes.func,
     selectBaseline: PropTypes.func,
-    selectOneBaseline: PropTypes.func,
     addSystemModalOpened: PropTypes.bool,
     fetchBaselines: PropTypes.func,
-    history: PropTypes.object,
+    tableType: PropTypes.string,
+    onClick: PropTypes.func,
     kebab: PropTypes.bool,
     createButton: PropTypes.bool,
     exportButton: PropTypes.bool,
     createBaselineModalOpened: PropTypes.bool,
-    hasSelect: PropTypes.bool,
-    hasSingleSelect: PropTypes.bool,
-    clearSelectedBaselines: PropTypes.func
+    onSelect: PropTypes.func,
+    clearSelectedBaselines: PropTypes.func,
+    columns: PropTypes.array
 };
 
 function mapStateToProps(state) {
     return {
         baselineDeleteLoading: state.baselinesTableState.baselineDeleteLoading,
-        baselineListLoading: state.baselinesTableState.baselineListLoading,
-        baselineTableData: state.baselinesTableState.baselineTableData,
-        addSystemModalOpened: state.addSystemModalState.addSystemModalOpened,
-        createBaselineModalOpened: state.createBaselineModalState.createBaselineModalOpened
+        addSystemModalOpened: state.addSystemModalState.addSystemModalOpened
     };
 }
 
 function mapDispatchToProps(dispatch) {
     return {
-        selectBaseline: (id, isSelected) => dispatch(baselinesTableActions.selectBaseline(id, isSelected)),
-        selectOneBaseline: (id, isSelected) => dispatch(baselinesTableActions.selectOneBaseline(id, isSelected)),
-        fetchBaselines: (params) => dispatch(baselinesTableActions.fetchBaselines(params)),
-        clearSelectedBaselines: () => dispatch(baselinesTableActions.clearSelectedBaselines())
+        clearSelectedBaselines: () => dispatch(baselinesTableActions.clearSelectedBaselines()),
+        fetchBaselines: (baselineType, params) => dispatch(baselinesTableActions.fetchBaselines(baselineType, params))
```

Comment:
Can both of these also be dropped thus dropping `mapDispatchToProps` ?

Reply:
That depends. I pass fetchBaselines to the helper function since I'm not sure how to dispatch an action from a helper without using mapDispatchToProps. Alternatively, I could drop it from BaselinesTable and pass the function in from 3 different parent Components.

Reply:
Ahh right, fair